chirp/drivers/uvk5.py

```diff
@@ -495,6 +495,13 @@ def _sayhello(serport):
         LOG.debug("Sending hello packet")
         _send_command(serport, hellopacket)
         o = _receive_reply(serport)
+
+        if o[0]==0x18 and o[1]==0x05:
+            LOG.warning("Radio is in firmware flash mode")
+            raise errors.RadioError(
+                    "This radio is in firmware flash mode (PTT + turn on).\n"
```

Comment:
There really shouldn't be newlines in exception messages, especially given a newline on its own isn't effective on each of the platforms we support. Please just make this a space and let the UI format it however it can.